api/dsr/v1/Access.md

```diff
@@ -0,0 +1,170 @@
+| *request.callbacks*          | no        | Array of [Callbacks](README.md#Callback)                                                                            |
+| *request.subject*            | yes       | The [Data Subject](README.md#Subject)                                                                               |
+| *request.claims*             | no        | Map containing additional claims that have been added via identity verification or other augmentation methods       |
+| *request.submittedTimestamp* | yes       | UNIX timestamp in seconds                                                                                           |
+| *request.dueTimestamp*       | yes       | UNIX timestamp in seconds                                                                                           |
+
+## Access Response / Status Event
+
+A successful response MUST include the `200 OK` response status code and a `AccessResponse` JSON object.
+
+When the status of Access Request has changed, a `AccessStatusEvent` event JSON object should be sent to all the callbacks
+specified in the request.
+
+The `event.results` and `events.documents` are merged with any cached results from previous events. New documents are
+added and existing documents are updated.
+
+Once the status is set to `completed`, `cancelled` or `denied`, then no further events will be accepted.
+
+The `Content-Type` MUST be `application/json`.
+
+```http request
+HTTP/1.1 200 OK
+Content-Type: application/json
+
+{
+  "apiVersion": "dsr/v1",
+  "kind": "AccessResponse",
+  "metadata": {
+    "uid": "22880925-aac5-42f9-a653-cb6921d361ff",
+    "tenant": "axonic"
+  },
+  "response": {
+    "status": "in_progress",
+    "expectedCompletionTimestamp": 123,
+    "results": [
+      {
+        "url": "https://example.com/results",
+        "headers": {
+          "Authorization": "Bearer $auth"
+        }
+      }
+    ]
+  }
+}
+```
+
+### Add context variables
+
+To add or update context variables, return the `claims` property.
+
+### Update data subject
+
+To alter the data subject details (such as including verified postal address), return either the complete Data Subject
+details in `subject` property or return an array of [JSON Patch](https://jsonpatch.com/) changes to be made to the Data
+Subject in the `subject` property.
+
+### Mutate identities
+
+* To add identities, return additional [Identities](README.md#Identity) in the `identities` property.
+* To replace an identity, return an [Identity](README.md#Identity) with the same `identitySpace` in the `identities` property.
+* To remove an identity, return an [Identity](README.md#Identity) with the same `identitySpace` in the `identities` property but without any `identityValue` property.
+
+### Add results
+
+To add results that should be made available to the Data Subject, return result documents in the `results` property.
```

Comment:
This is a json object now right? Not callback like example? https://github.com/ketch-com/ketch-forwarder/pull/11/files#diff-72ba22185acb3032dda0910db9ad336580b5c954ea6376c0480084b65fc74ab3R112-R119